web/app/js/components/Navigation.jsx

```diff
@@ -425,55 +427,31 @@ class NavigationBase extends React.Component {
 
         <Divider />
 
-        <MenuList>
-          <Button
-            variant="contained"
-            className={classes.namespaceChangeButton}
-            size="large"
-            onClick={this.handleNamespaceMenuClick}>
-            { formattedNamespaceName }
-            <ArrowDropDownIcon />
-          </Button>
-          <Menu
-            anchorEl={anchorEl}
-            open={this.state.namespaceMenuOpen}
-            keepMounted
-            onClose={this.handleNamespaceMenuClick}>
-
-            <MenuItem>
-              <InputBase
-                id="namespace-filter-textfield"
-                className={classes.inputBase}
-                value={namespaceFilter}
-                onChange={this.handleFilterInputChange}
-                placeholder="Select namespace..."
-                autoFocus />
-            </MenuItem>
-
-            <Divider />
-
-            <MenuItem
-              value="all"
-              onClick={e => this.handleNamespaceChange(e, "_all")}>
-                  All Namespaces
-            </MenuItem>
-
-            {namespaces.map(ns => (
-              <MenuItem
-                onClick={e => this.handleNamespaceChange(e, ns.name)}
-                key={ns.name}>
-                {ns.name}
-              </MenuItem>
-              ))}
-          </Menu>
-          <NamespaceConfirmationModal
-            open={showNamespaceChangeDialog}
-            selectedNamespace={selectedNamespace}
-            newNamespace={newNamespace}
-            handleDialogCancel={this.handleDialogCancel}
-            handleConfirmNamespaceChange={this.handleConfirmNamespaceChange} />
-
-        </MenuList>
+        <Autocomplete
```

Comment:
And we can apply styles like this and remove the `styles` prop:

```
          size="small"
          classes={{
            root: classes.namespaceChangeButton,
            inputRoot: classes.namespaceChangeButtonInputRoot,
            input: classes.namespaceChangeButtonInput,
            popupIndicator: classes.namespaceChangeButtonPopupIndicator,
          }}
```

Reply:
👍